packages/core/strapi/src/utils/postinstall.ts

```diff
@@ -15,7 +9,7 @@ const machineID = require('./machine-id');
 try {
   if (
     process.env.npm_config_global === 'true' ||
-    JSON.parse(process.env.npm_config_argv).original.includes('global')
+    JSON.parse(process.env.npm_config_argv ?? '').original.includes('global')
   ) {
```

Comment:
Empty string will throw anyway, if this is just to make typescript happy I'd use `as any` so it's more clear and doesn't change the actual execution at all.

Reply:
Sure I want it to throw if it's undefined so I can use any here